rbpf-cli/src/main.rs

```diff
@@ -1,6 +1,6 @@
 fn main() {
     println!(
         r##"rbpf-cli is replaced by solana-ledger-tool run subcommand.
-Please, use 'solana-ledger-tool run --help' for more information."##
+Please, use 'solana-ledger-tool program run --help' for more information."##
```

Comment:
nit: Since you're already in there, this should just be `solana-ledger-tool program --help`